changelogs/fragments/9-semantic-markup-roles.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - "Add support for semantic markup in roles (https://github.com/ansible-community/antsibull-docs-parser/pull/9)."
+breaking_changes:
+  - "Modify ``LinkProvider.plugin_option_like_link`` signature to include optional string argument ``entrypoint`` after ``plugin`` (https://github.com/ansible-community/antsibull-docs-parser/pull/9)."
```

Comment:
`optional string` seems confusing. The type is `str | None`, but the argument still needs to be passed.